src/base/memory_monitor_common.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * memory_monitor_common.hpp - common structures for cubrid memory monitoring module
```

Comment:
```suggestion
 * memory_monitor_common.hpp - common structures for memory monitoring module
```

Reply:
The previous comment is Nit :D

I think we don't mention 'CUBRID' prefix because everybody knows it.

Reply:
The change has been made accordingly :)